staging/src/k8s.io/sample-apiserver/artifacts/example/ns.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: wardle
+spec:
+  finalizers:
+  - kubernetes
```

Comment:
why this finalizer?

Reply:
Good question, I', not sure why I added it. I suppose it can be dropped then.

Reply:
They'll at least get added by default.